apps/explorer/tailwind.config.js

```diff
@@ -6,6 +6,7 @@ const vegaCustomClasses = require('../../libs/tailwindcss-config/src/vega-custom
 module.exports = {
   content: [
     join(__dirname, 'src/**/*.{js,ts,jsx,tsx}'),
+    'libs/ui-toolkit/src/utils/shared.ts',
```

Comment:
question (non-blocking): trying to understand why this is needed, but can't wrap my head around it. this is supposed to be just a js array of class names for form elements, right? why do the apps need to know about this file?

Reply:
I initially added this file to only the ui-toolkit tailwind config expecting the shared classes to be bundled in and used by the apps. However it doesn't work unless the app specific tailwind config includes it.